pom.xml

```diff
@@ -8,7 +8,27 @@
     <artifactId>JavaBootcampEY</artifactId>
     <version>1.0-SNAPSHOT</version>
 <dependencies>
-
+    <dependency>
+        <groupId>com.opencsv</groupId>
+        <artifactId>opencsv</artifactId>
+        <version>4.1</version>
+    </dependency>
+    <!-- Maven exercise
+    <dependency>
+        <groupId>org.apache.logging.log4j</groupId>
+        <artifactId>log4j-core</artifactId>
+        <version>2.12.0</version>
+    </dependency>-->
+    <dependency>
+        <groupId>org.slf4j</groupId>
+        <artifactId>slf4j-api</artifactId>
+        <version>2.0.7</version>
+    </dependency>
+        <dependency>
```

Comment:
this dependency is used for replace original Log4j and delegate all logs into Slf4j. this sytyation exists when we have old library  that uses log4j but in our application we using different logger like Logbag.